core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -466,9 +506,13 @@ contract PricelessPositionManager is FeePayer {
 
     /**
      * @dev These internal functions are supposed to act identically to modifiers, but re-used modifiers
-     * unneccessarily increase contract bytecode size
+     * unnecessarily increase contract bytecode size.
      * source: https://blog.polymath.network/solidity-tips-and-tricks-to-save-gas-and-reduce-bytecode-size-c44580b218e6
      */
+    function _isContractStateOpen() internal view {
```

Comment:
nit: I think this name is kinda strange for a function that throws if the state is not open. Some suggested alternatives: `_onlyContractStateOpen()`, `_onlyOpen()`, `_requireOpen()`, etc. I know others are named this way, but I think they're kinda strange too :). Another option would be to make this function return a bool and put a require around the bool in the modifier (although I'm not sure that's as gas efficient).

Reply:
I followed the same convention we have with `_isPreExpiration` `_isCollateralizedPosition` and `_isPostExpiration`. I agree it's a bit kunky; my personal preference is to pull these back into modifiers such that all of these kinds of checks are done in the modifiers, even if it increases the bytecode by a small amount. There is currently an issue open around this change #984

Reply:
Hmmm, I'm not suggesting that we stop using helper functions. Rather, I'm suggesting that we rename the helper functions to be more indicative of what they do. IMO, `_is*` doesn't sound like a function that reverts if the condition is false.

Reply:
I understand. I will refactor all of them to follow the `_only*` notation.